Nativing/accounts/models.py

```diff
@@ -1,3 +1,87 @@
 from django.db import models
+from django.contrib.auth.models import PermissionsMixin, AbstractBaseUser, BaseUserManager
 
-# Create your models here.
+
+class CustomAccountManager(BaseUserManager):
+    def create_user(
+        self,
+        name, 
+        email, 
+        nickname,
+        date_of_birth,
+        user_image,
+        user_gender,
+        password=None
+    ):
+        user = self.model(
+            name=name,
+            email=email, 
+            nickname=nickname, 
+            date_of_birth=date_of_birth,
+            user_image=user_image,
+            user_gender=user_gender,
+        )
+        user.set_password(password)
+        user.is_superuser = False
+        user.save(using=self._db)
+        return user
+
+    def create_superuser(
+        self,
+        name, 
+        email, 
+        nickname, 
+        password
+    ):
+        user = self.model(
+            email=email, 
+            name=name,
+            nickname=nickname, 
+            password=password)
+        user.is_admin = True
+        user.is_superuser = True
+        user.save(using=self._db)
+        return user
+
+
+class Gender:
+    """
+    Gender
+    - Distinguish Male, Female
+    """
+    MALE = 'male'
+    FEMALE = 'female'
+
+    GENDER_TYPES = [
+        (MALE, '남성'),
+        (FEMALE, '여성'),
+    ]
+
+
+class User(AbstractBaseUser, PermissionsMixin):
+    name = models.CharField(max_length=30, null=True, blank=True)
+    email = models.EmailField(
+        max_length=255,
+        unique=True,
+    )
+    nickname = models.CharField(max_length=15, blank=True, null=True, unique=True)
+    joined_on = models.DateTimeField(auto_now_add=True)
+    withdrew_at = models.DateTimeField(blank=True, null=True)
+    date_of_birth = models.CharField(max_length=32, blank=True, help_text='생년월일 6자리')
+    user_image = models.ImageField(upload_to='images/', blank=True, null=True)
+    user_gender = models.CharField(max_length=10, choices=Gender.GENDER_TYPES)
+    user_age = models.PositiveIntegerField(blank=True, null=True)
+    is_login = models.BooleanField(default=False)
+    is_admin = models.BooleanField(default=False)
```

Comment:
```suggestion
    is_admin = models.BooleanField(default=False)
    is_superuser = models.BooleanField(default=False)
```
`AbstractBaseUser`에 `is_superuser` property가 원래 상속이 되나요?
저는 일단 저번에 커스텀 유저 모델 제작할 때는 선언을 해주었어서.. 혹시나 해서 추가합니다. 
없어도 되는거면 빼구 위에 일반 유저 생성할 떄 is_superuser = False 넣는 거로 하면 될 거 같아요.